frontend/src/pages/ProjectSettings/ErrorFiltersForm/ErrorFiltersForm.tsx

```diff
@@ -0,0 +1,99 @@
+import { FieldsBox } from '@components/FieldsBox/FieldsBox'
+import { CircularSpinner, LoadingBar } from '@components/Loading/Loading'
+import Select from '@components/Select/Select'
+import { useEditProjectMutation, useGetProjectQuery } from '@graph/hooks'
+import { namedOperations } from '@graph/operations'
+import { useParams } from '@util/react-router/useParams'
+import { message } from 'antd'
+import clsx from 'clsx'
+import React, { useEffect, useState } from 'react'
+
+import commonStyles from '../../../Common.module.scss'
+import Button from '../../../components/Button/Button/Button'
+import styles from './ErrorFiltersForm.module.scss'
+
+export const ErrorFiltersForm = () => {
+	const { project_id } = useParams<{
+		project_id: string
+	}>()
+	const [errorFilters, setErrorFilters] = useState<string[]>([])
+	const { data, loading } = useGetProjectQuery({
+		variables: {
+			id: project_id!,
+		},
+		skip: !project_id,
+	})
+
+	const [editProject, { loading: editProjectLoading }] =
+		useEditProjectMutation({
+			refetchQueries: [
+				namedOperations.Query.GetProjects,
+				namedOperations.Query.GetProject,
+			],
+		})
+
+	const onSubmit = (e: { preventDefault: () => void }) => {
+		e.preventDefault()
+		if (!project_id) {
+			return
+		}
+		editProject({
+			variables: {
+				id: project_id!,
+				error_filters: errorFilters,
+			},
+		}).then(() => {
+			message.success('Updated error filters!', 5)
+		})
+	}
+
+	useEffect(() => {
+		if (!loading) {
+			setErrorFilters(data?.project?.error_filters || [])
+		}
+	}, [data?.project?.error_filters, loading])
+
+	if (loading) {
+		return <LoadingBar />
+	}
+
+	return (
+		<FieldsBox id="errors">
+			<h3>Error Filters</h3>
+
+			<form onSubmit={onSubmit} key={project_id}>
+				<p>Enter REGEXP patterns to filter out errors.</p>
+				<div className={styles.inputAndButtonRow}>
+					<Select
```

Comment:
Can we tidy up the empty state?

![Screenshot 2023-04-03 at 9 17 38 AM](https://user-images.githubusercontent.com/58678/229553752-a82c003b-e285-41ad-a6a5-5bc56825da51.png)

Reply:
updated